aidial_adapter_bedrock/llm/model/llama/v3.py

```diff
@@ -40,7 +40,7 @@ def encode_header(message: BaseMessage) -> str:
 
 def encode_message(message: BaseMessage) -> str:
     ret = encode_header(message)
-    ret += message.content.strip()
+    ret += message.text_content
```

Comment:
I guess, if we used strip before, let's be consistent
```python
message.text_content.strip()
```

Reply:
Indeed, fixed